File: src/main/java/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/MessageEntityFilterValidator.java
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl;

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.CustomValidator;
import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator;
import io.github.oleksivio.telegram.bot.api.model.objects.std.MessageEntity;

public class MessageEntityFilterValidator extends FilterValidator<MessageEntity> {

    public MessageEntityFilterValidator(CustomValidator<MessageEntity> validator) {
        super(MessageEntity.class, validator);
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/annotation/validator/impl/MessageEntityFilterValidator.kt
package io.github.oleksivio.telegram.bot.api.model.annotation.validator.impl

import io.github.oleksivio.telegram.bot.api.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.std.MessageEntity
import io.github.oleksivio.telegram.bot.core.controller.handler.check.Validator

class MessageEntityFilterValidator(validator: Validator<MessageEntity>) : FilterValidator<MessageEntity>(
    MessageEntity::class, validator)
